CHANGELOG.rst

```diff
@@ -1,3 +1,11 @@
+0.9 (Unreleased)
+----------------
+
+* Drop pep8 requirement and replace with pycodestyle.
+* Support Flake8 3.0 (alongside Flake8 2.X).
+* Drop Python2.6 compatibility.
+* Refactor code structure, fixing bugs in the process.
```

Comment:
Good point, I've updated the changelog. It is #76 and #49, #50 I think are fixed.